sunpy/io/tests/test_cdf.py

```diff
@@ -23,3 +25,83 @@ def test_read_cdf():
     assert col.unit == u.Unit("1 / (cm2 MeV s sr)")
     # Check that fillvals are replaced by NaN
     assert np.sum(np.isnan(col)) == 189
+
+
+def test_generic_timeseries_columns():
```

Comment:
All we need to check is that the column of the timeseries which should have nans in, has nans in it.

Reply:
We should also check that the original code example passes now as well.